src/main/java/com/faforever/client/preferences/LoginPrefs.java

```diff
@@ -6,24 +6,17 @@
 import javafx.beans.property.StringProperty;
 import lombok.extern.slf4j.Slf4j;
 
-import java.net.InetAddress;
-import java.net.NetworkInterface;
-import java.net.SocketException;
-import java.net.UnknownHostException;
 import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 
 @Slf4j
+
 public class LoginPrefs {
 
   private static final String KEY;
 
   static {
-    try {
-      KEY = new String(NetworkInterface.getByInetAddress(InetAddress.getLocalHost()).getHardwareAddress(), StandardCharsets.US_ASCII);
-    } catch (SocketException | UnknownHostException e) {
-      throw new UnsupportedOperationException("No socket available");
-    }
+    KEY = System.getProperty("user.name");
```

Comment:
The `static {}` block was only necessary because the code threw an exception before. Now you can set the value directly where you declare KEY.